src/metabase/server/routes/index.clj

```diff
@@ -78,33 +78,35 @@
         (throw (Exception. message e))))))
 
 (defn- template-parameters
```

Comment:
Click on "Minimize comments" to review

<img width="378" height="494" alt="Image" src="https://github.com/user-attachments/assets/0f9fb093-1383-41cf-bd83-494844fa43e8" />